src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -956,15 +956,39 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <comment>{0} is the severity of the vulnerability</comment>
   </data>
   <data name="VulnerabilitySeverity_Critical" xml:space="preserve">
-    <value>Critical</value>
+    <value>critical</value>
```

Comment:
It's supposed to be lowercase according to the picture in the design spec. I mistakenly had made it upper-case because at one point we had talked about it being that way, but the picture in the design spec is lowercase so I fixed it here.